pom.xml

```diff
@@ -0,0 +1,393 @@
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>role-strategy</artifactId>
+							<version>${role-strategy.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>saferestart</artifactId>
+							<version>${saferestart.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>saml</artifactId>
+							<version>${saml.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>scm-api</artifactId>
+							<version>${scm-api.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>script-security</artifactId>
+							<version>${script-security.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>ssh-credentials</artifactId>
+							<version>${ssh-credentials.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>timestamper</artifactId>
+							<version>${timestamper.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>token-macro</artifactId>
+							<version>${token-macro.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins.workflow</groupId>
+							<artifactId>workflow-step-api</artifactId>
+							<version>${workflow-step-api.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+					</artifactItems>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>com.googlecode.maven-download-plugin</groupId>
+				<artifactId>download-maven-plugin</artifactId>
+				<version>1.3.0</version>
+				<executions>
+					<execution>
+						<id>fetch-marathon-deploy</id>
```

Comment:
This is in the Jenkins update center (version 1.3.1) so we can treat it as any of the other plugins now and not host it in our S3 bucket